development/build/index.js

```diff
@@ -0,0 +1,133 @@
+const gulp = require('gulp')
+const gulpZip = require('gulp-zip')
+const livereload = require('gulp-livereload')
+const del = require('del')
+const mkdirp = require('mkdirp')
+// const imagemin = require('gulp-imagemin')
+const baseManifest = require('../../app/manifest/_base.json')
+
+const { createTask, taskSeries, taskParallel, runTask } = require('./task')
+
+const createManifestTasks = require('./manifest')
+const createScriptTasks = require('./scripts')
+const createStyleTasks = require('./styles')
+const createStaticAssetTasks = require('./static')
+
+const browserPlatforms = [
+  'firefox',
+  'chrome',
+  'brave',
+  'opera',
+]
+
+//
+// tasks
+//
+
+async function clean () {
+  await del(['./dist/*'])
+  await Promise.all(browserPlatforms.map(async (platform) => {
+    await mkdirp(`./dist/${platform}`)
+  }))
+}
+
+// browser reload
+
+createTask('reload', function devReload () {
+  livereload.listen({ port: 35729 })
+})
+
+
+const staticTasks = createStaticAssetTasks({ livereload, browserPlatforms })
+const manifestTasks = createManifestTasks({ livereload, browserPlatforms })
```

Comment:
Nit: `livereload` isn't used for the manifest tasks
```suggestion
const manifestTasks = createManifestTasks({ browserPlatforms })
```